VisualizationBase/src/ViewItemManager.h

```diff
@@ -52,11 +56,22 @@ class VISUALIZATIONBASE_API ViewItemManager
 		ViewItem* currentViewItem();
 		void removeAllViewItems();
 
+		void saveView(ViewItem* view, Model::TreeManager* manager) const;
+		ViewItem* loadView(QString name, QPoint position = QPoint(-1, -1));
+
 		QVector<QVector<ViewItem*>> viewItems() const;
+		QList<ViewItem*> viewItemsAsList() const;
+
+		void cleanupRemovedItem(Visualization::Item* removedItem);
```

Comment:
This is not exactly a public method. Better make it private and declare whoever is calling it as a friend. friend declarations should appear on top of the private section in a class.